mongodb-advanced/components/people-component.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default {
+  controller: Ctrl,
+  templateUrl: 'mongodb-advanced/people-component.html'
+};
+
+function Ctrl($scope, exPeopleService) {
+  const self = this;
+  self.showMyModal = false;
+
+  exPeopleService.getPeople(people => {
+    self.savedPeople = people;
+    $scope.$apply('refreshData');
+  });
+
+  self.modalClosed = () => {
+    exPeopleService.getPeople(people => {
+      self.savedPeople = people;
+      $scope.$apply('refreshData');
```

Comment:
When you update the service (see below) to just return $http, you will no longer need $scope.apply() in these situations. 

$scope.apply() has very specific and limited use.  Broadly speaking you'll use $scope.apply() only when interacting with components that do not return a $q style promise.